source/fahrplanparser.d

```diff
@@ -33,11 +21,18 @@ public:
 * data is expected to contain valid XML as returned by queries sent to http://mobile.defas-fgi.de/beg/.
 */
 
-auto parsedFahrplan(in string data)
+auto parsedFahrplan(in string data, in long walkingDelay = 0, in SysTime currentTime = Clock.currTime)
```

Comment:
We should only work with `DateTime` and not `SysTime`. To get the current time as `DateTime`, simply use a cast:
```D
auto parsedFahrplan(in string data, in long walkingDelay = 0, in DateTime currentTime = cast(DateTime) Clock.currTime)
```
```